modules/test/playwright/tests/journal-web/pages/JournalPage.ts

```diff
@@ -267,4 +267,30 @@ export class JournalPage {
 
 		await expect(tagCheckbox).toBeHidden();
 	}
+
+	async highlightStructure(
```

Comment:
Couldn't we move this test to integration in JournalDisplayContextTest?

Maybe just checking if the url has this extra parameter could be enough to test the fix (and integration test are more robust than functional ones)

Reply:
Yes, make sense. I am changing it from funcional to integration test. 

thanks!